mobile/src/components/chat-conversation/components/message-form/message-form.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react';
+
+import { AppColor } from '~/common/enums/enums';
+import { ChatMessageCreateRequestBodyDto } from '~/common/types/types';
+import { Icon, Input, Pressable, View } from '~/components/common/common';
+import { useAppForm, useCallback, useFocusEffect } from '~/hooks/hooks';
+import { chatMessageCreateArguments } from '~/validation-schemas/validation-schemas';
+
+import { getDefaultMessagePayload } from './helpers/helpers';
+import { styles } from './styles';
+
+type Props = {
+  chatId: string | null;
+  chatOpponentId: number;
+  onSubmit: (payload: ChatMessageCreateRequestBodyDto) => void;
+};
+
+const MessageForm: FC<Props> = ({ chatId, chatOpponentId, onSubmit }) => {
+  const { control, errors, handleSubmit, reset } =
+    useAppForm<ChatMessageCreateRequestBodyDto>({
+      defaultValues: getDefaultMessagePayload(chatOpponentId, chatId),
+      validationSchema: chatMessageCreateArguments,
+    });
+
+  const hitSlop = { top: 5, bottom: 5, left: 5, right: 5 };
+
+  useFocusEffect(
+    useCallback(() => {
+      return () => {
+        reset(getDefaultMessagePayload(chatOpponentId, chatId));
+      };
+    }, []),
+  );
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.input}>
+        <Input
+          placeholder="Type a message"
+          name="message"
+          control={control}
+          errors={errors}
+        />
+      </View>
+      <Pressable
+        onPress={handleSubmit(onSubmit)}
+        hitSlop={hitSlop}
+        style={styles.button}
```

Comment:
Maybe if we remove background and make icon blue and bigger as in telegram it will look more simple?

Reply:
I've just tried too make it look like the rest buttons in the app

Reply:
| variant 1  | variant 2 |
| ------------- | ------------- |
| ![Screenshot_1](https://user-images.githubusercontent.com/89215185/188877058-04df6cdd-5aa3-4670-91ae-dde0f6eabae1.png)  | ![Screenshot_2](https://user-images.githubusercontent.com/89215185/188877077-4b8531db-0f73-4e19-9aec-c30796b96940.png)  |

Reply:
I would prefer first option for action buttons